README.md

```diff
@@ -19,8 +19,14 @@ This simple python file will tell you the team's next game, date, team they're p
 - Bayern Munich
 
 ## Instructions:-
-Simply download and run the fixtures.py file and watch the program display the schedule.
+1. Clone the repository
+2. Install the required packages using `pip install -r requirements.txt`
```

Comment:
FWIW, I incorporated this into my PR here: https://github.com/alonelysaber/Fixture-Calendar/pull/3/files

Reply:
k

Reply:
Not really sure how adding argparse with minimal documentation allows him to learn better, but feel free to use your own issue/PR.
Take care!

Reply:
You're welcome to suggest some changes to any of the PRs! That's part of the benefit of collaboration on GitHub, IMO! :) I'm certainly open to constructive feedback.

Do you feel there is less documentation in my PRs than previously? Please let me know!

Reply:
I appreciate your input!
Yeah, I just feel like the your changes that followed my initial PR were very minimal other than the argparse package and felt like the implementation that I had was just manipulated slightly in your own PR.
But regardless, feel free to keep yours if you feel that it is better as I closed my PR.
Take care!

Reply:
I was actually going to suggest all the changes on your PR, but it was becoming too difficult to try and re-factor on the webui, so I brought **_your_** branch into my IDE to modify. 😄 The changes were large enough (needed to split this PR into two) and changed the original execution of the program that I opted to separate them into different PRs. My hope was **_not_** for you to close your PR, but rather, for you to see yet another way to refactor the changes beyond what you had already done (great job, by the way!).

Perhaps it would help if I outlined the changes I made on top of your PR? List below:

### In #3

1. Update the docs further to advise to use a [Python Virtual Environment](https://docs.python.org/3/tutorial/venv.html) - this is a best-practice as to not cause people to `pip install` dependencies into their main python installation. This can cause a lot of future trouble for folks.
2. New-line the individual teams in the `TEAMS` list, as to make subsequent changes easier - the way it currently was would cause the entire line to show as changed when reviewing a PR to add/remove a new team; having the teams on individual lines makes the PRs much easier to quickly see.
3. Remove duplicate call to `get(url)` in `team_name()` and `parser()` methods - this causes the person to make two network requests for the same resource, and is not needed (why waste the bandwidth, slow down the program, or cause 2x the number of requests needed? 😄 ).
4. Add [typing](https://docs.python.org/3/library/typing.html) - this is a good practice, and allows tools such as `mypy` or IDEs to perform linting to catch issues/bugs early on.
5. In-line unnecessary variables such as `date_time_comp`
6. Convert to use [`f-strings`](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings) where possible.
7. Remove unnecessary `if/else` (line 34-36 in this PR) and opt for an in-line ternary operator for simplicity
8. Simplify the printing of all teams to use a `for` loop (2 lines, more pythonic and easier to read/understand).

### In https://github.com/ChrisCarini/Fixture-Calendar/pull/1

1. Use argparse to introduce a new flag for the functionality you added (really neat idea, by the way!). Typically when modifying programs, it's good to leave the default behavior and introduce something to invoke the new code path, in this case, the prompt. Perhaps @alonelysaber really just wants to run a script and see everything, without entering anything.
2. Extract the prompt logic into a stand-alone method that returns the index of the team in the `TEAMS` array - this simplifies the `int(team) - 1` that was going on and makes the subsequent code in the main block easier to reason about (_'prompt for team, then print that team selection'_).
3. Dynamically generate the prompt menu based on the teams specified in the `TEAMS` list - the way implemented in this PR would cause @alonelysaber (or, whoever) to need to change both the `TEAMS` list, as well as the prompt found on line 55 - this is prone to human error (What if the new team is put at the beginning of the list, but the option is added to the end of the list? This would cause everything in the menu to print the wrong thing!).
4. Improve the error message when an invalid input is entered - just telling someone 'invalid input' is a great start, but it's even more helpful if you advise them **what** a valid input consists of, in this case, a number between 1 and 'x'.

Given all that, I'd actually love to see you re-open your PR and shepard it through, incorporating some of these changes as you / @alonelysaber see fit (I intentionally did not do all the refactorings I could think of, some left untouched would be (a) adding python docstrings to the methods, (b) adding some small unit tests, (c) handling a default input on the menu (ie, user just hits enter w/o entering a number)). I see you are in your final year of college - collaborating with people on GitHub was not something I did early in my career, but wish had done. I'd encourage you to take the two PRs I opened as constructive feedback and an opportunity to learn, ask questions (both to myself, and others), and to reopen your PR. 😄

Reply:
I appreciate your thoughtful feedback, re-reading my responses in this conversation I feel like I sounded a bit harsh, so I apologize for that.
And correct, I am still learning about GitHub and how these collaboration features work even in my last part of my education.
I appreciate the explanations and the useful feedback, but I do feel that now that you provided more documentation of your implementation, it makes a lot more sense.
I wish you the best!

Reply:
> I appreciate your thoughtful feedback, re-reading my responses in this conversation I feel like I sounded a bit harsh, so I apologize for that.

No problem - don't sweat it!

> And correct, I am still learning about GitHub and how these collaboration features work even in my last part of my education.

You're lucky! :) My education covered all of near-zero-percent of the collaborative aspect of software development, let alone about working with total strangers online. Again, falls into the bucket of "wish I had done this in the earlier part of my career" - I applaud you for doing so! :)

> I appreciate the explanations and the useful feedback, but I do feel that now that you provided more documentation of your implementation, it makes a lot more sense.

No problem - to be fair, I should have likely included the lists above in my PR descriptions. That's my fault, sorry about that!

> I wish you the best!

Thank you, you as well! Happy to chat more if you'd like any guidance or have questions on post-graduation life - it's an exciting time for you! :)